libgcc/config/i386/cygwin.S

```diff
@@ -107,14 +103,6 @@ __alloca:
 	pushl	4(%eax)
 	ret
 	cfi_endproc()
-#elif defined(__aarch64__)
-	cfi_startproc()
-___chkstk:
-__alloca:
-	// TODO
```

Comment:
What code is AArch64 using now? Without changes in this file, x64 instructions would generated for AArch64.

Reply:
aarch64 has implementation for stack probing, it is used for mingw32 and it looks like it works for cygwin, internal tests should be compiled to validate it. i386 chkstk has been removed from the configuration and this file is not used.

Reply:
OK, thank you for the explanation.